web/urls.py

```diff
@@ -9,11 +9,12 @@
 	url(r'^about/$', views.about, name='pages-about'),
 	url(r'^projects/$', views.projects, name='pages-projects'),
 	url(r'^members/$', views.members, name='pages-members'),
+	url(r'^member-page/(?P<user_id>[0-9]+)/$', views.member_page),
 	url(r'^events/$', views.events, name='pages-events'),
 	url(r'^gallery/$', views.gallery, name='pages-gallery'),
 	url(r'^login/$', views.login, name='pages-login'),
 	url(r'^links/$', views.links, name='pages-links'),
-	url(r'^edituser/$', views.edituser, name='pages-edituser'),
+	url(r'^edituser/(?P<user_id>[0-9]+)/$', views.edituser),
```

Comment:
So I see you actually removed name for this URL, can I ask about the reasoning behind this decision?